### PR TITLE
Fixes #38. Add a description field to the UI for tasksi

### LINKED ISSUE #38: Add description field for analyses & workflows
Allow authors to write a brief description for their code blocks, which will then be displayed to prospective users.

### PR DESCRIPTION
**Pull request by zachmullen**

### CODE REVIEW

**Comment by jeffbaumes** [comment]
Looking very good. Could you also display the input descriptions on the "Setup and run" dialog on this branch? Perhaps using this style

http://getbootstrap.com/css/#forms-help-text

Displaying the type:format of each input would be nice too.

The analysis description on the main panel could possibly also use this style but not sure how it would look - use your best judgment.

Couple more things we could do:
- Hovering over inputs/outputs/task in the workflow editor could show description text if there is any.
- The "Setup and run" dialog should contain all help text - including repeating the task description near the top and enumerating outputs with their descriptions (and type/format) somewhere in the dialog.

**Comment by zachmullen** [comment]
@jeffbaumes I added the task descriptions and input descriptions and types to the setup and run dialog.

**Comment by jeffbaumes** [comment]
This is looking very nice. Two more things:
* Since Romanesco does auto-conversion between formats, the actual input format is an implementation detail that does not need to be exposed in the Setup and run dialog, so I think it should be changed to just include the type.
* Include a list of output names, types (again no formats), and descriptions under the main task description in the Setup and run dialog.

With these two changes I think this branch will be ready. I'll make a separate issue for exposing descriptions in the workflow editor.

**Comment by zachmullen** [comment]
@jeffbaumes ptal

File: app/src/views/AnalysisSetupView.js [hunk 1]
```diff
@@ -119,9 +119,19 @@
         },
 
         render: function () {
-            this.inputsView.collection.set(this.model.get('meta').analysis.inputs);
-            this.inputsView.render();
-            this.$('.analysis-setup-title').text(this.model.get('meta').analysis.name);
+            var analysis = this.model.get('meta').analysis;
+            console.log(analysis);
```
Comment (jeffbaumes) [inline app/src/views/AnalysisSetupView.js]: Remove debug output.

**Comment by jeffbaumes** [comment]
Looks great! After debug statement is removed LGTM.

**Pull request merged**